plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -56,6 +61,15 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
         }
     }
 
+    private fun getJepVersionFromVersionsFile(conclaveBuildDir: Path): String {
```

Comment:
@shamsasari I have struggled with this one. Do you know if there is a better mechanism to retrieve the `jep_version` from `versions.gradle` while we are in the Gradle Plugin code?